### DISCUSSION TITLE
Improvement idea: 3DPrintColorizer and dual extrusion (PLA-PVA)

### DISCUSSION THREAD

**Discussion by DatMinh94** [Sakati84/3DPrintColorizer#90] [Ideas]
Hello @Sakati84 and everyone. I need to print colorized anatomical model for surgical planning at my hospital. We don't have a big budget to buy a polyjet printer, so your idea is our very savior. Furthermore, we think multicolor print + dual extrusion (PLA-PVA) is feasible with FDM printer. We are trying to use a dual extruder printer. One nozzle will do the color printing using your genius method and the other nozzle will print soluble support material. Can you modify the plug in so that the last slot or "virtual nozzle" in Cura is always reserved for the second nozzle to print PVA? We very much appreciate your job. That would be perfect if we can print PVA and multicolor with a FDM machine.
![1](https://user-images.githubusercontent.com/78602717/154487387-21ececc1-3a0b-4b27-b625-2a113777b334.jpg)

**Comment by Nathan22211** [comment]
I'll have to look but I think all we need to do is replace the regex expression that removes T commands to only remove 1-7 and then subtract 7 from 8 and above. Though Klipper might handle this better by changing the T macro to T8
